play/src/front/Stores/MediaStore.ts

```diff
@@ -324,6 +324,10 @@ export const inJitsiStore = writable(false);
 export const inBbbStore = writable(false);
 export const isSpeakerStore = writable(false);
 
+export const doNotDisturbStore = writable(false);
+export const backInAMomentStore = writable(false);
+export const busyStore = writable(false);
+
```

Comment:
Instead of these 3 stores (that could be in conflict), I wonder if we should not have a single store called `requestedStatusStore` that contains the store the user wants.

Note it can be different from the status the user is granted in the end (because it can be overridden by a silent zone, etc...)